src/app/components/header/header.component.html

```diff
@@ -0,0 +1,9 @@
+<p>
+  <mat-toolbar>
+    <div class="nav">
+      <a class="link" routerLink="" routerLinkActive="active">Super Quiz</a>
```

Comment:
please change class naming in whole app to BEM approach.
So this one would be header__link